Bond/Bond+Arrays.swift

```diff
@@ -663,50 +663,50 @@ private class DynamicArrayDeliverOnProxy<T>: DynamicArray<T> {
     super.init([])
     
     bond.willInsertListener = { [unowned self] array, i in
-      dispatch_async(queue) { [weak self] in
```

Comment:
We should still be able to optionally unwrap self even with `unowned`?  `self?.dispatchWillInsert(i)`